src/gc/os.d

```diff
@@ -105,9 +157,22 @@ else static if (is(typeof(mmap)))  // else version (GC_Use_Alloc_MMap)
     {
         return munmap(base, nbytes);
     }
+
+    /**
+     * Possible results for the wait_pid() function.
+     */
+    enum WRes
+    {
+     DONE, /// The process has finished successfully
+     RUNNING, /// The process is still running
+     ERROR /// There was an error waiting for the process
+    }
```

Comment:
Just one question, why is this `WRes` defined twice?

Reply:
Sorry, I am just learning magit and I forgot to stage some lines.
I made another  commit. I can squash if needed but I don't think it makes sense at this point.

Reply:
I can't define const HAVE_FORK in the same way sociomantic does because I get:
`src/gc/os.d(145): Error: variable `gc.os.HAVE_FORK` conflicts with variable gc.os.HAVE_FORK at src/gc/os.d(116)`